src/powershell/FinOpsToolkit.psm1

```diff
@@ -0,0 +1,274 @@
+data LocalizedData
+{
+    # culture="en-US"
+    ConvertFrom-StringData -StringData @'
+    DeployFinOpsHub = Deploying hub from template '{0}' to resource group '{1}'.
+    ExpandingZip = Expanding zip '{0}'.
+    FoundAsset = Found asset '{0}'.
+    FoundLatestRelease = Found latest release '{0}'.
+    NewDirectory = Creating directory '{0}'.
+    NotLoggedIn = Not logged into Azure. Launching 'Login-AzAccount'.
+    VersionNotFound = Could not find version '{0}' of FinOpsHub. Run 'Get-FinOpsHubVersions' for available versions.
```

Comment:
Versioning will be at a toolkit level. Let's up-level this...
```suggestion
    VersionNotFound = Could not find FinOps toolkit version '{0}'. Run 'Get-FinOpsToolkitVersions' for available versions.
```

Reply:
done